package.json

```diff
@@ -131,7 +131,7 @@
 	},
 	"peerDependencies": {
 		"@guardian/ab-core": "^7.0.1",
-		"@guardian/consent-management-platform": "^13.11.1",
+		"@guardian/consent-management-platform": "13.12.0",
```

Comment:
Hi @vlkerag, you're missing a `^` here I think?

Reply:
Thank you!